### ISSUE TITLE
AttributeError: readonly attribute

### ISSUE THREAD

**Issue by reubano** [k-bx/python-semver#49]
```
python --version
Python 3.2.5 (b2091e973da6, Oct 19 2014, 18:29:55)
[PyPy 2.4.0 with GCC 4.6.3]
```

```
Collecting semver<3.0.0,>=2.2.1 (from pkutils<0.14.0,>=0.13.4->-r dev-requirements.txt (line 19))
  Downloading semver-2.7.5.tar.gz
    Traceback (most recent call last):
      File "<string>", line 20, in <module>
      File "/tmp/pip-build-8u5mmw/semver/setup.py", line 2, in <module>
        import semver as package
      File "semver.py", line 68, in <module>
        VersionInfo.__doc__ = """
    AttributeError: readonly attribute
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
    
      File "<string>", line 20, in <module>
    
      File "/tmp/pip-build-8u5mmw/semver/setup.py", line 2, in <module>
    
        import semver as package
    
      File "semver.py", line 68, in <module>
    
        VersionInfo.__doc__ = """
    
    AttributeError: readonly attribute
```

**Comment by reubano** [comment]
This is the [bad commit](https://github.com/k-bx/python-semver/commit/7afb3805db85f6127d571863e94a75e72e8f795a).

**Comment by k-bx** [comment]
Fixed in https://github.com/k-bx/python-semver/pull/50

**Comment by k-bx** [comment]
Released under 2.7.6
